project/CONTRIBUTING.md

```diff
@@ -72,9 +72,30 @@ php-cs-fixer fix --verbose
 
 #### The documentation
 
-The documentation is mostly written with the `rst` format. You can test the doc rendering with the `make docs` command.
+The documentation is mostly written with the `rst` format,
+and can be found in the `{{ docs_path }}` directory.
+You can test the doc rendering with the `make docs` command,
+but to do this, you will need [Sphinx](http://www.sphinx-doc.org/en/stable/).
+Just like php dependencies can be managed with Composer,
+python dependencies can be managed with [pip](https://pip.pypa.io/en/stable/installing/).
+To get sphinx, simply run the following command.
+
+```bash
+pip install --requirement Resources/doc/requirements.txt --user
+```
+
+Some python binaries should be downloaded in `~/.local/bin`,
+[modify your `$PATH` environment variable](http://www.linfo.org/path_env_var.html)
+so that it contains this path and then, from the root of the project, run `make docs`
+
+If `make docs` is successful, you should be able to see your modifications:
+
+```bash
+$YOUR_FAVORITE_BROWSER {{ docs_path }}/_build/html/index.html
```

Comment:
I don't think so since it's not an url : 

```
open Resources/doc/_build/html/index.html
Couldn't get a file descriptor referring to the console
```